op-chain-ops/genesis/setters.go

```diff
@@ -152,9 +152,9 @@ func MigrateDepositHashes(hh *hardhat.Hardhat, db vm.StateDB) error {
 // This is an optimization to make calling them cheaper. This should only
 // be used for devnets.
 func SetPrecompileBalances(db vm.StateDB) {
-	for i := 0; i <= 9; i++ {
+	for i := 0; i <= 256; i++ {
```

Comment:
First 256 would be `<= 255` or `< 256` since 0 counts. Now we visit 0 twice.

Reply:
Updated.

Reply:
Whoops, good catch